utilities/policy_utilities.mak

```diff
@@ -116,9 +119,9 @@ $(O)/key_utility.o: $(US)/key_utility.cc $(INC_DIR)/support.h $(INC_DIR)/certifi
 	@echo "compiling key_utility.cc"
 	$(CC) $(CFLAGS) -c -o $(O)/key_utility.o $(US)/key_utility.cc
 
-#$(US)/certifier.pb.cc $(I)/certifier.pb.h: $(CERT_SRC)/certifier.proto
-#	$(PROTO) -I$(S) --cpp_out=$(US) $(CERT_SRC)/certifier.proto
-#	mv certifier.pb.h $(I)
+$(CERT_SRC)/certifier.pb.cc $(I)/certifier.pb.h: $(CERT_SRC)/certifier.proto
+	$(PROTO) -I$(S) --proto_path=$(CERT_SRC) --cpp_out=$(CERT_SRC) $(CERT_SRC)/certifier.proto
+	mv $(CERT_SRC)/certifier.pb.h $(I)
```

Comment:
This _seems_ correct; i.e. the generated `.h` file will be in `certifier/src` sub-dir.